### DISCUSSION TITLE
Any plans for the new version release

### DISCUSSION THREAD

**Discussion by am2222** [duckdb/duckdb-wasm#1795] [Ideas]
Hi,
I have noticed there has been several bug fixes and improvements to the library but the latest version on the `npm` is from 8 months ago. I was wondering if there is any schedules to release new npm version.

**Comment by jonathonadams** [comment]
I know this is a couple of months old, but If you haven't come across it yet, the `next` tag is currently being used for all new releases. i.e. `npm i @duckdb/duckdb-wasm@next`

See [here](https://www.npmjs.com/package/@duckdb/duckdb-wasm/v/1.28.1-dev278.0?activeTab=versions)
